src/app/plugins/checkclasses.plugin.ts

```diff
@@ -48,25 +47,18 @@ export class CheckClassesPlugin extends Plugin {
       return;
     }
 
-    const chansForMessage: string[] = this._adaptToChanMessageString(chansContainingUser);
-    chansForMessage.forEach((m) =>
-      message.reply(`User is registered for \`${chansContainingUser.length}\` classes:\n\`${m}\``)
-    );
-  }
-
-  private _adaptToChanMessageString(userChans: GuildChannel[]): string[] {
-    const chanNames = userChans.map((c) => c.name);
-
-    //If its not longer than the char limit, send it
-    const toString = chanNames.join(' | ');
-    if (toString.length < this._MAX_CHAR_LIMIT) {
-      return [toString];
-    }
-
-    const middle = chanNames.length / 2; //Find the middle element
-    const splitChans = [chanNames.slice(0, middle), chanNames.slice(middle)]; //Split the classes into 2 equal groups
+    const chansForMessage = chansContainingUser.map((c) => c.name).join(' | ');
 
-    //For each group, join to string
-    return splitChans.map((chans) => chans.join(' | '));
+    this.container.messageService
+      .sendTextMessage(
+        message,
+        `User is registered for \`${chansContainingUser.length}\` classes:\n\`${chansForMessage}\``,
+        { reply: true, delimiter: ' ' }
```

Comment:
![image](https://user-images.githubusercontent.com/39809146/113513529-341daf80-9538-11eb-9853-ec2ba333ee5c.png)
This doesn't wrap in code across multiple messages correctly

Reply:
I'll add a title option, that goes before the header on the first message.